src/components/dictionaryConcepts/components/helperFunction.js

```diff
@@ -1,2 +1,221 @@
 export const getTotal = () => localStorage.getItem('total_concepts');
 export const getUsername = () => localStorage.getItem('username');
+
+export const classes = [
+  'Drug',
+  'Diagnosis',
+  'Finding',
+  'Anatomy',
+  'Question',
+  'LabSet',
+  'MedSet',
+  'ConvSet',
+  'Misc',
+  'Symptom',
+  'Symptom-Finding',
+  'Specimen',
+  'Misc-Order',
+  'Workflow',
+  'State',
+  'Program',
+  'Aggregate-Measurement',
+  'Indicator',
+  'Health-Care-Monitoring-Topics',
+  'Radiology-Imaging-Procedure',
+  'Frequency',
+  'Pharmacologic-Drug-Class',
+  'Units-of-Measure',
+  'Organism',
+  'Drug-form',
+  'Medical-supply',
+  'InteractSet',
+  'Test',
+  'Procedure',
+];
+
+export const locale = [
+  { value: 'aa', label: 'Afar [aa]' },
```

Comment:
Is this any different from this? https://github.com/openmrs/openmrs-ocl-client/blob/6ad889fcddf3acf5de5bdd3c1b318f4f6b05266b/src/components/dashboard/components/dictionary/common/Languages.js

Reply:
No, I actually sent the code to @EleisonC because he needed it too.

Reply:
We will eventually settle for one, this will probably be done in a fixture ticket.

Reply:
Can't you just use his?

Reply:
As at the time I implemented it he didn't have that file, neither was the file in our codebase. However, I will rebase now and refactor my code to use his version since his PR has been merged.